ambry-cloud/src/main/java/com.github.ambry.cloud/HelixVcrStateModel.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2019 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.cloud;
+
+import org.apache.helix.NotificationContext;
+import org.apache.helix.model.Message;
+import org.apache.helix.participant.statemachine.StateModel;
+import org.apache.helix.participant.statemachine.StateModelInfo;
+import org.apache.helix.participant.statemachine.Transition;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * {@link StateModel} to use when the VCR participants register to Helix. The methods are callbacks
+ * that get called within a participant whenever its state changes in Helix.
+ */
+@StateModelInfo(initialState = "OFFLINE", states = {"LEADER", "STANDBY"})
```

Comment:
Can we use constants for the state strings?

Reply:
I wish to, but couldn't find a good place to define constants. (`@StateModelInfo` is out of this class)
Since this is not a common changed class, I feel it's OK to use plainText.